contracts/HashRegistrarSimplified.sol

```diff
@@ -130,8 +130,7 @@ contract Registrar {
     //   Auction -> Reveal
     //   Reveal -> Owned
     //   Reveal -> Open (if nobody bid)
-    //   Owned -> Forbidden (invalidateName)
-    //   Owned -> Open (releaseDeed)
+    //   Owned -> Open (releaseDeed or invalidateName)
```

Comment:
then just leave it as is. I bet we'll eventually use that space again in the future